plugins/common/lambda-rules-registration.cpp

```diff
@@ -0,0 +1,70 @@
+#include "lambda-rules-registration.hpp"
```

Comment:
You could put the contents of this file in the header, making this header-only, that's the approach taken by other similar classes.

Reply:
Done